templates/kubeflow/features.html

```diff
@@ -160,7 +160,7 @@ <h2>Operational dashboards</h2>
         </div>
       </div>
       <div class="u-fixed-width">
-        <img src="https://assets.ubuntu.com/v1/9f6916dd-k8s-prometheus-light.png?w=1040" alt="" width="1040" />
+        <img src="https://assets.ubuntu.com/v1/9f6916dd-k8s-prometheus-light.png" alt="" width="1040" />
```

Comment:
This image is 300kb can it be reduced without losing legibility?

Reply:
Oddly, lots of things didn't make it smaller; however, converting it to jpeg did help... it is about 144kb now.